bin/yk-config

```diff
@@ -197,21 +197,21 @@ if [ $# -eq 0 ]; then
 fi
 
 case $1 in
-    debug|release);;
-    *) 1>&2 echo "unknown mode: $1\n"
+    debug*|release*);;
+    *) 1>&2 echo "Profile '$1' does not start with 'debug' or 'release'.\n"
```

Comment:
Should it be: `debug|release|release-with-debug)`? That way we reject (for example) `debug-with-donuts`.

Reply:
The PR deliberately allows `debug-with-donuts` in case you want to do so!

Reply:
Ah, in case you define a target in your local config? gotcha.

Reply:
You've got it :)